mineinabyss-components/src/main/kotlin/com/mineinabyss/components/music/Song.kt

```diff
@@ -0,0 +1,13 @@
+package com.mineinabyss.components.music
+
+import com.charleskorn.kaml.YamlComment
+import kotlinx.serialization.Serializable
+import kotlin.time.Duration
+
+@Serializable
+data class Song(
+    val key: String,
+    val duration: Duration,
```

Comment:
Wonder if we could make a custom Duration handler that lets u chain durations?
Like this would require us to specify in seconds, cant do say "2m32s"
Minor thing but

Reply:
Swapped these to serialize via DurationSerializer, which also supports formatting like "1m40s"